Rovers/Rovers.tex

```diff
@@ -0,0 +1,388 @@
+% Below are examples of using nomenclature for math symbols and constants or units
+\nomenclature{$\dot{m}$}{Mass flow rate
+  \nomunit{\,\si{\kilo\gram\per\second}}}
+\nomenclature{$c$}{Speed of light
+ \nomunit{\,\SI{2.9979e8}{\meter\per\second}}}
+\printnomenclature{}
+
+% The sections included here are required. Additional sections and subsections may be added as necessary.
+\section{Introduction}
+\label{sec:introduction}
+  % The introduction is a place to give background and context before diving into the subject matter.
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{R}{obotics}  and by extension rovers are a tremendously important
+part of space exploration. 
+This is also an area that RIT
+Space Exploration has very little experience with project-wise.
+The purpose of this project is to assess and assert the capability
+of RIT SPEX regarding the construction and fabrication of a mock-rover and then begin construction. 
+As this is a new area to RIT SPEX, prototyping and experimenting is necessary to most effectively accomplish this goal. 
+This project will look at the unknowns, technical challenges, project management, and member skills of RIT Space Exploration in regard to a University Rover Competition (URC) capable rover. 
+The URC is hosted by The Mars Society annually. 
+It is very competitive and requires a very complex and expensive rover that is currently out of scope for SPEX. 
+This rover would need to be capable of full autonomous navigation, soil collection and analysis, and precise robotics. 
+This project is the first step in building the skills for such a team.
+
+
+\section{Primary Objective}
+\label{sec:primary-obj}
+  % At the end of the day, whether the project ``succeeds'' or ``fails'' is judged against the objectives it sought to meet.
+  % Note that results that contradict expectations/hypotheses are not failures if the scientific \& engineering methods are followed along the way.
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+
+The goal of this project is delivering a functioning mock rover.
+We define mock rover as a functioning rover built to specifications
+for a mock mission as it will not be launched. The rover
+will allow the team to develop skills and knowledge to build a
+more feature completer rover in the future. These skills include
+rover fabrication and design, autonomous navigation, electrical
+system design, rover testing, computer vision, and robotics.
+The rover will be testing using events designed like
+the URC but scaled down to fit the scope of this rover. We
+will be using time as a metric as well as binary metrics for
+completion of the parts of the competition. This includes ’does
```

Comment:
for properly formatted apostrophes and quotes, you have to use the ` ` ` character. for example
```
Thomas continued his monologue, ``And then Phil told me something amazing about `style` and `tables` and it changed my life forever!``
```